README.md

```diff
@@ -0,0 +1,163 @@
+![Stream Cli](https://i.imgur.com/H8AScTq.png)
+
+# Stream CLI
+
+Stream's Command Line Interface (CLI) makes it easy to create and manage your [Stream](https://getstream.io) apps directly from the terminal. Currently, only Chat is supported; however, the ability to manage Feeds will be coming soon.
+
+> **Note**: The latest version of Node (v14.x) is required for this CLI. If you are looking for a way to manage multiple Node.js environments, please look at [nvm](https://github.com/nvm-sh/nvm).
+
+
+
+# 🏗 Installation
+
+The Stream CLI is built with Go and we compile it into a single binary so you don't need any prerequisite available on your computer. You can simply download it and put it to your `$PATH`.
+
+The releases are available in [GitHub Releases](https://github.com/GetStream/stream-cli/releases).
+
+On MacOS:
+```shell
+$ curl -L https://github.com/GetStream/stream-cli/releases/latest/download/darwin -o $HOME/stream-cli
```

Comment:
Let's make the binary available on common package manager (homebrew, apt, ...)

Reply:
Can be definitely done for a next iteration

Reply:
yeah, homebrew/apt is a bit away for now.